.travis.yml

```diff
@@ -4,8 +4,9 @@ services:
 script:
 - docker build --target test --tag my-test-image .
 - docker run my-test-image todo_app/items_test.py    
-- docker run -e TOKEN -e KEY -e TOID -e DOINGID -e DONE -e CLIENT my-test-image  todo_app/e2e_test.py 
+- docker run -e TOKEN -e KEY -e TOID -e DOINGID -e DONE -e CLIENT -e GITHUBID -e WEBAPPLICATIONCLIENT -e CLIENTSECRET -e TODOBOARD -e DOINGBOARD -e DONEBOARD -e CONNECTIONSTRING -e WEBHOOK_URL my-test-image  todo_app/e2e_test.py 
 - docker build --target production --tag my-production-image .
+- bash webhook.sh
```

Comment:
This should go in your deployment step. It definitely needs to run after you push your production image to Docker Hub. It's also fine to get rid of the Heroku steps now that we're deploying to Azure.

Reply:
Updated, thank you